src/cmd/src/options.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+use common_telemetry::logging::LoggingOptions;
+use datanode::datanode::DatanodeOptions;
+use frontend::frontend::FrontendOptions;
+use meta_srv::metasrv::MetaSrvOptions;
+
+pub enum ConfigOptions {
+    Datanode(Box<DatanodeOptions>),
+    Frontend(FrontendOptions),
```

Comment:
Any reason for not having `Box` here?

Reply:
Just clippy without warnings. I think it may be that most of the internal members of `FrontendOptions` are `Option<>`, and clippy cannot accurately calculate the size.
Do I need to change it to `Box<FrontendOptions>`?

Reply:
Yes, I suggest to change all these type to boxed for consistency